src/main/java/me/drex/itsours/command/Command.java

```diff
@@ -72,12 +70,41 @@ boolean hasPermission(ServerCommandSource src, String permission) {
 
     public AbstractClaim getClaim(CommandContext<ServerCommandSource> ctx) throws CommandSyntaxException {
```

Comment:
It would much nicer if you implemented the get logic in a separate method and kept this just as a shortcut to that.